.changelog/unreleased/state-breaking/provider/2106-query-blocks-until-next-epoch.md

```diff
@@ -0,0 +1,2 @@
+- Add a query to get the blocks until the next epoch begins
```

Comment:
I wonder if this is `state-breaking` because this just introduces the query and the new logic:
```
	if k.BlocksUntilNextEpoch(ctx) == k.GetBlocksPerEpoch(ctx) {
```
should be equivalent to what we had before (?)

Reply:
Yeah it is equivalent... I thought for a moment this would be statebreaking because of the new query, but its just a query, so Ill delete this changelog entry